src/test/java/com/easypost/TestUtils.java

```diff
@@ -24,29 +26,37 @@ public enum ApiKey {
 
     private static final String API_KEY_FAILED_TO_PULL = "couldNotPullApiKey";
     private static final String CASSETTES_PATH = "src/test/cassettes";
-    private static final java.util.List<String> HEADER_CENSORS = new ArrayList<String>() {{
-        add("Authorization");
-        add("User-Agent");
-    }};
-    private static final List<String> QUERY_CENSORS = new ArrayList<String>() {{
-        add("card[cvc]");
-        add("card[number]");
-    }};
-    private static final List<String> BODY_CENSORS = new ArrayList<String>() {{
-        add("api_keys");
-        add("client_ip");
-        add("credentials");
-        add("key");
-        add("keys");
-        add("phone_number");
-        add("phone");
-        add("test_credentials");
-    }};
-    private static final List<CensorElement> BODY_ELEMENTS_TO_IGNORE_ON_MATCH = new ArrayList<CensorElement>() {{
-        // Timezone difference between machines causing failure on replay
-        add(new CensorElement("createdAt", false));
-        add(new CensorElement("updatedAt", false));
-    }};
+    private static final java.util.List<String> HEADER_CENSORS = new ArrayList<String>() {
```

Comment:
Question: How did this all get reformatted? Your IDE or the `make format` command?

Reply:
This project doesn't have a `make format`, if someone wanted to add one that'd be great! IDE did this. It should be pulling in our style guide as a part of it.